csrank/tests/test_ranking.py

```diff
@@ -55,24 +76,35 @@ def tunable_parameters(cls):
 
 @pytest.fixture(scope="module")
 def trivial_ranking_problem():
-    rand = np.random.RandomState(123)
-    x = rand.randn(100, 5, 1)
+    random_state = np.random.RandomState(123)
+    x = random_state.randn(200, 5, 1)
     y_true = x.argsort(axis=1).argsort(axis=1).squeeze(axis=-1)
     return x, y_true
 
 
 def test_fate_object_ranker_fixed(trivial_ranking_problem):
+    tf.set_random_seed(0)
+    os.environ["KERAS_BACKEND"] = "tensorflow"
+    np.random.seed(123)
+    for ranker_name in object_rankers.keys():
+        loss = 0.0
+        rtol = 1e-2
+        atol = 1e-8
+        assert object_ranker_fixed(trivial_ranking_problem, ranker_name=ranker_name, loss=loss, rtol=rtol, atol=atol)
+
+
+def object_ranker_fixed(trivial_ranking_problem, ranker_name=FATE_RANKER, loss=0.0, rtol=1e-2,
```

Comment:
Good idea, and I think we can even simplify the code more by using [pytest.mark.parametrize](https://docs.pytest.org/en/latest/parametrize.html#parametrizemark).

Reply:
Done